### PR TITLE
Added movie Hobbs & Show

### PR DESCRIPTION
**Pull request by nayane95**
Added the movie Hobbs & Show to the 2019 folder

### CODE REVIEW

File: movies/2019/hobbs-&-show.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Hobbs & Show",
+    "year": 2019,
+    "runtime": 136,
+    "categories": [
+      "action"
+
```
Comment (leonlafa) [inline movies/2019/hobbs-&-show.json]: Remove white space 😄

**Comment by lukeloresch** [comment]
"Hobbs and Shaw" is the correct movie title spelling.

File: movies/2019/hobbs-&-show.json (hunk 1 above)
Comment (nayane95) [reply movies/2019/hobbs-&-show.json]: @leonlafa  thanks

**Comment by nayane95** [comment]
> 
> 
> "Hobbs and Shaw" is the correct movie title spelling.

corrected

File: movies/2019/hobbs-&-shaw.json [hunk 2]
```diff
@@ -1,10 +1,9 @@
 {
-    "name": "Hobbs & Show",
+    "name": "Hobbs & Shaw",
```
Comment (leonlafa) [inline movies/2019/hobbs-&-shaw.json]: ahhh! Good find! 😄

File: movies/2019/hobbs-&-shaw.json [hunk 3]
```diff
@@ -1,10 +1,9 @@
...
     "year": 2019,
     "runtime": 136,
     "categories": [
-      "action"
-      
+      "action"
```
Comment (leonlafa) [inline movies/2019/hobbs-&-shaw.json]: Nice one!

**Pull request merged**